Sources/SwiftSyntaxBuilder/ConvenienceInitializers.swift

```diff
@@ -13,6 +13,26 @@
 @_spi(RawSyntax) import SwiftParser
 @_spi(RawSyntax) import SwiftSyntax
 
+// MARK: - ArrayElementList
+
+extension ArrayElementListSyntax {
+  public init(expressions: [ExprSyntaxProtocol]) {
+    let elements = expressions.enumerated().map { index, expression in
+      let element = ArrayElementSyntax(expression: expression)
+      return index != (expressions.count - 1) ? element.with(\.trailingComma, .commaToken()) : element
```

Comment:
This will override any potentially already existing trailing comma (and trivia associated with it). You should be able to use `element.ensuringTrailingComma()` instead.

Also, I’d prefer an `if` to the ternary expression. I just think it reads a lot more easily.